modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -368,4 +372,5 @@ you-do-not-have-the-permission-to-view-this-form=You do not have the permission
 you-have-entered-invalid-json=You have entered invalid JSON.
 you-need-to-be-signed-in-to-view-this-form=You need to be signed in to view this form.
 you-will-also-delete-fields-with-this-row-are-you-sure-you-want-delete-it=You will also delete fields with this row. Are you sure you want delete it?
-your-information-was-successfully-received-thanks-for-fill-out=Your information was successfully received. Thank you for filling out the form.
\ No newline at end of file
+your-information-was-successfully-received-thanks-for-fill-out=Your information was successfully received. Thank you for filling out the form.
```

Comment:
key deveria ter `filling` e não apenas `fill`:

your-information-was-successfully-received-thank-you-for-filling-out-the-form

Ou podes simplificar a key e deixar apenas `your-information-was-successfully-received`